godo.go

```diff
@@ -223,13 +227,72 @@ func addOptions(s string, opt interface{}) (string, error) {
 	return origURL.String(), nil
 }
 
+// configureRetryableClient will create a retryableclient with default behavior
+// if no godo client is passed. It will then convert the retryableclient to a *http.Client.
+func configureRetryableClient(c *Client, httpClient *http.Client) *http.Client {
+	retryableClient := retryablehttp.NewClient()
+
+	if c == nil {
+		retryableClient.RetryMax = defaultRetryMax
+		retryableClient.RetryWaitMin = time.Duration(defaultRetryWaitMin * float64(time.Second))
+		retryableClient.RetryWaitMax = time.Duration(defaultRetryWaitMax * float64(time.Second))
```

Comment:
By not setting a `nil` logger here, we get the upstream default which logs every request. This is a change for `NewFromToken` that might be pretty unexpected. e.g. If some one has built a command line utility, they likely won't want this logging directly to stdout.